features/OCM_rule_feedback.feature

```diff
@@ -0,0 +1,240 @@
+Feature: Rule feedback ability
+
+  Scenario: Displaying rule feedback controls on OCM UI
+    Given console.redhat.com is accessible
+      And user U1 is already logged in into console.redhat.com
+      And user U1 switch to "OpenShift" page in console.redhat.com (OCM UI)
+      And user U1 is part of account (organization) with at least one cluster C1
+      And the cluster C1 sent data to the external data pipeline
+      And 1 potential security issue has been identified (ie. one security recommendation has been generated by CCX pipeline)
+      And no voting has been made across the account
+     When user U1 selects "Clusters" item from the menu
+     Then new page with list of clusters should be displayed
+      And the page should contain at least one cluster
+      And button "Create cluster" should be made visible
+      And button "Register cluster" should be made visible
+     When user U1 click on cluster name
+     Then a page with detailed information about the selected cluster should be displayed
+      And at least four tab headers should be displayed on that page
+          | Tab name         |
+          | Overview         |
+          | Monitoring       |
+          | Insights Advisor |
+          | Support          |
+     When user U1 clicks on tab named "Insights Advisor"
+     Then new page should be displayed saying "1 potential issue identified"
+      And the page should contains several sections
+      And "Recommendations by category section" should display the following data
```

Comment:
`"Recommendations by category section"` should be `"Recommendations by category" section`.